Scripts/MovementSystem/MovableObject.cs

```diff
@@ -3,11 +3,21 @@
 
 public partial class MovableObject : Node
 {
-    [Export] float speed = 10;
+    [Export] float speed = 200;
     [Export] RigidBody2D rb;
 
+
     public void Move(Vector2 direction)
     {
-        // TODO: implement moving of the rigidbody
+        if (rb == null) return;
+
+        var dir = direction;
+        if (dir.LengthSquared() > 1f) dir = dir.Normalized();
+
+        rb.LinearVelocity = dir * speed;
+        rb.Sleeping = false;
     }
+
+    public Node2D GetBodyNode2D() => rb as Node2D;
```

Comment:
tohle klidně může vracet RigidBody a může se to jmenovat třeba GetRb nebo GetRigidBody2D